apis/falcon/v1alpha1/falconconfig_types.go

```diff
@@ -0,0 +1,52 @@
+/*
+Copyright 2021 CrowdStrike
+*/
+
```

Comment:
The per-file license headers are not needed.

Ref: https://github.com/CrowdStrike/Open-Source-Policy#project-licensing

Double checked with CrowdStrike legal last week. In reality they never were. Nobody sure how that came to be, so we explicitly documented it is not needed.